package.json

```diff
@@ -20,6 +20,7 @@
     "test-redux-watch": "npm run test-redux -- --watch",
     "test": "npm run test-styles && npm run test-redux",
     "postinstall": "npm run build-prod",
+    "start-dev": "node ./devserver",
```

Comment:
This is unnecessary as `yarn dev` will run everything :)

Reply:
Hi @Dinoshauer - thanks for getting back!  Hope you had some good time off :-)

And yes, sorry, forgot to update this bit - discovered I had a local Dynamodb process squatting on port 8000 which was throwing me.  Will update...